server/service/handler.go

```diff
@@ -454,6 +454,10 @@ func attachFleetAPIRoutes(r *mux.Router, svc fleet.Service, config config.FleetC
 	he.WithAltPaths("/api/v1/osquery/log").
 		POST("/api/osquery/log", submitLogsEndpoint, submitLogsRequest{})
 
+	// orbit authenticated endpoints
+	oe := newOrbitAuthenticatedEndpointer(svc, logger, opts, r, apiVersions...)
+	oe.POST("/api/_version_/fleet/orbit/flags", getOrbitFlagsEndpoint, orbitGetConfigRequest{})
```

Comment:
not a flags only anymore, `/api/_version_/fleet/orbit/config` maybe?

Reply:
as a note: we should also standardize the naming for the endpoint function and the request type, right? `getOrbitFlagsEndpoint` => `getOrbitConfigEndpoint` and `orbitGetConfigRequest` => `getOrbitConfigRequest`

and a nit: I find it confusing that this is a `POST` but all the methods are actions associated with `GET`, is this to send the token?

Reply:
thanks! if we're getting stuff, maybe a note will suffice, but it's a nit, I wouldn't worry too much about it.

another way to tackle this if we want an homogeneous interface for all request methods (not for now) would be to send the token in a header `X-Orbit-Token: xyz`